src/plugins/data/public/ui/query_string_input/language_switcher.tsx

```diff
@@ -52,10 +95,25 @@ export function QueryLanguageSwitcher(props: Props) {
       className="euiFormControlLayout__append kqlQueryBar__languageSwitcherButton"
       data-test-subj={'switchQueryLanguageButton'}
     >
-      {props.language === 'lucene' ? luceneLabel : kqlLabel}
+      {props.language === 'lucene' ? luceneLabel : props.language === 'kuery' ? kqlLabel : eqlLabel}
```

Comment:
nit: `props.` appears 10 times within this function. consider destructuring `props`